pkg/controller/podgc/gc_controller.go

```diff
@@ -31,41 +32,52 @@ import (
 	clientset "k8s.io/client-go/kubernetes"
 	corelisters "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/util/workqueue"
 	"k8s.io/component-base/metrics/prometheus/ratelimiter"
 
 	"k8s.io/klog"
 )
 
 const (
+	// gcCheckPeriod defines frequency of running main controller loop
 	gcCheckPeriod = 20 * time.Second
+	// quarantineTime defines how long unknown nodes wait before check
```

Comment:
Please, please be more elaborative here :) Be more precise about what is being checked for unknown nodes.

Reply:
Done